docsets/Scikit/docset.json

```diff
@@ -1,10 +1,10 @@
 {
     "name": "scikit-learn",
-    "version": "1.1.2",
+    "version": "1.2.2",
     "archive": "scikit-learn.tgz",
```

Comment:
Please update the archive at this path. You added `scikit-learn-1.2.2.tgz` instead.